modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/NumberInputTest.ts

```diff
@@ -0,0 +1,76 @@
+import { FocusTools, Keys, UiControls } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarElement, SugarShadowDom } from '@ephox/sugar';
+import { TinyAssertions, TinyDom, TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.themes.silver.throbber.NumberInputTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    toolbar: [ 'undo', 'fontsizeinput', 'redo' ]
+  }, []);
+
+  it('TINY-9429: plus and minus should increase and decrease font size of the current selection', () => {
+    const editor = hook.editor();
+    editor.setContent('<p>abc</p>');
```

Comment:
Maybe feed it a font size with a fixed value first then use plus since it would otherwise in theory mean that plus is just applying the value that is in the input not reading and increasing the current value.

Reply:
changed